omemo/encryption.py

```diff
@@ -92,4 +92,20 @@ def migrate(dbConn):
                                      END TRANSACTION;
                                  """ % (create_table))
 
+    if user_version(dbConn) < 2:
+        delete_dupes = """ DELETE FROM identities WHERE _id not in (
+                            SELECT MIN(_id)
+                            FROM identities
+                            GROUP BY
+                            recipient_id, public_key
+                            );
+                        """
+
```

Comment:
there cant be any duplicates in the future once this query was run, cause i revised the function who writes into that table to check for jid AND key, before writing.
the function is saveIdentity()

i hope i understood the question right